packages/sources/README.md

```diff
@@ -109,6 +110,7 @@ This document was generated automatically. Please see [Master List Generator](..
 - [poa](./poa/README.md)
 - [polygon](./polygon/README.md)
 - [por-address-list](./por-address-list/README.md)
+- [reduce](./reduce/README.md)
```

Comment:
I don't think this should be included here, right?

Reply:
It seems this was automatically added after i tested master list generation locally and `reduce` is included because somehow i have that folder with only dist in it. In theory this should be regenerated once merged but for safety i'm gonna remove it.